pleskdistup/common/tests/phptests.py

```diff
@@ -76,3 +76,123 @@ def test_no_outdated(self):
             php.get_outdated_php_handlers(version.PHPVersion("5.0")),
             []
         )
+
+
+class TestGetHandlersByCondition(unittest.TestCase):
+    def test_lover_then_version_7_1(self):
+        self.assertEqual(
+            sorted(php.get_php_handlers_by_condition(lambda php: php < version.PHPVersion("7.1"))),
+            [
+                "plesk-php52-cgi",
```

Comment:
If this test is for `get_php_handlers_by_condition()`, it shouldn't contain a fixed list of handlers. Otherwise, it becomes a test for combination of  `get_php_handlers_by_condition()`, `get_known_php_versions()` and `get_php_handlers()`, which isn't a unit test.

Reply:
Yes, it's expected. I'm trying to see how the entire module works here. For example, if the list of known PHP versions changes, I want to make sure the test detects it.
Otherwise, I would only be checking how one line with a condition call works, which doesn't make much sense to me.